src/cf.data.pre

```diff
@@ -4413,6 +4413,41 @@ DOC_START
 		cache_dir rock /ssd3 ... max-size=99999
 DOC_END
 
+NAME: paranoid_hit_validation
+COMMENT: time-units-small
+TYPE: time_nanoseconds
+DEFAULT: 0
+DEFAULT_DOC: validation disabled
+LOC: Config.paranoid_hit_validation
+DOC_START
+		Controls whether Squid should perform paranoid validation of cache entry
+		metadata integrity every time a cache entry is hit. Squid bugs
+		notwithstanding, this low-level validation should always succeed. Each
+		failed validation results in a cache miss, a BUG line reported to cache.log,
+		and the invalid entry marked as unusable (and eventually purged from the
+		cache).
+
+		Squid can only validate shared cache memory and rock cache_dir entries.
+
+		Positive paranoid_hit_validation values enable validation and approximate
+		the maximum time that Squid is allowed to spend validating a single cache
+		hit. Huge values greater or equal to 1 day are considered as no limitation:
```

Comment:
"Huge" is unnecessary bloat when the next thing is a more precise definition of the range.